src/Runner.Common/HostContext.cs

```diff
@@ -343,6 +343,12 @@ public string GetConfigFile(WellKnownConfigFile configFile)
                         ".runner");
                     break;
 
+                case WellKnownConfigFile.MigratedRunner:
+                    path = Path.Combine(
+                        GetDirectory(WellKnownDirectory.Root),
+                        ".runner_migrated");
```

Comment:
we are going to reuse `.credentials_migrated`.

Reply:
we already delete the `.credentials_migrated` as part of https://github.com/actions/runner/pull/513/files#diff-f965276d94b7d1ede0f58a5e1341987e3cf976566f942e5f96e23717ec52140c